components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -13415,6 +13505,16 @@ public String getAPILevelTier(String apiUUID, String revisionUUID) throws APIMan
         return null;
     }
 
+    public ThrottlingLimit getAPIThrottlingLimit(String apiUUID, String revisionUUID) throws APIManagementException {
+
+        try (Connection connection = APIMgtDBUtil.getConnection()) {
+            return getAPIThrottlingLimit(connection, apiUUID, revisionUUID);
+        } catch (SQLException e) {
+            handleException("Failed to retrieve Connection", e);
```

Comment:
Rename error based on the new table name and method signature.

Reply:
since we are returning a single string, thought of maintaining the method name as it is in this case.